src/slices/main/fab/Fab.css

```diff
@@ -1,7 +1,34 @@
 .fab {
-    position: fixed;
+    position: absolute;
     right: 40px; /* Must be set in pixels because Popover's viewportPadding is in pixels */
     bottom: 40px;
+    opacity: 1;
+
+    &.bounceIn {
+        animation: bounce-in 450ms linear;
```

Comment:
Should we change this (and other occurencies) from `450ms` to `0.45s` to keep the consistency with other components?

Reply:
Agreed. Done!